Base/res/apps/VideoPlayer.af

```diff
@@ -0,0 +1,7 @@
+[App]
+Name=Video Player
+Executable=/bin/VideoPlayer
+Category=Graphics
```

Comment:
Maybe we should rename the `Sound` category to `Multimedia` and put it there? It's not _exactly_ a graphics program.

Reply:
Sure, that makes sense to me :)

Reply:
After speaking in Discord, it seems that Media might be a more appropriate option.